js/data/line_bucket.js

```diff
@@ -27,6 +27,11 @@ var EXTRUDE_SCALE = 63;
 var COS_HALF_SHARP_CORNER = Math.cos(75 / 2 * (Math.PI / 180));
 var SHARP_CORNER_OFFSET = 15;
 
+// The maximum distance along a line that can be stored in the buffers.
+var MAX_LINE_DISTANCE = Math.pow(2, 14);
+var LINE_DISTANCE_SCALE = 1 / 2;
```

Comment:
Please add a comment clarifying the unit of this value (even if it is "extent units")